Mods/ModVarietyPack/Defs/ThingDefs/Apparel_MVP.xml

```diff
@@ -2952,10 +2952,10 @@ And the surgical mask prevents unwanted bacteria entering the vic.. patient</des
 
 
 
-  <!--ArchNem© Ski™ goggles-->
+  <!--ArchNem� Ski� goggles-->
   <ThingDef ParentName="ApparelBase">
     <defName>Apparello_Brimsk</defName>
-    <label>ArchNem© Ski™ goggles</label>
+    <label>ArchNem� Ski� goggles</label>
```

Comment:
Something happened here with the © symbol after your commit. 
Make sure you save in UTF-8 not something else. It gets messed up in game otherwise. 
My guess is that was because the .gitattributes messing with the text format. But not sure